site/content/internal/onboarding/manager-guide.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Manager Guide
+date: 2017-11-07T14:28:35-05:00
+subsection: Onboarding
+weight: 30
+---
+
+### Mentor For The Day
+- Schedule a 1-1 each day bewteen new staff and team members (including QA and PM) and other devs who would be relevant to mee with for the first two weeks
+- Meet with the new staff member for 5-10 minutes everyday for the first week or so to make sure everything is going well
```

Comment:
5-10 mins or 20-30min ?  should also include something like "hangout and get to know each other"

Reply:
Meant for the manager to meet with the new staff member everyday in addition to the mentor meeting, not for the 1-1 between the mentor and new staff. I can update the text to be clearer about that